build.gradle.kts

```diff
@@ -42,12 +42,6 @@ dependencies {
 
   implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
 
-  implementation("javax.annotation:javax.annotation-api:1.3.2")
-  implementation("javax.xml.bind:jaxb-api:2.3.1")
```

Comment:
not sure how I didn't notice these old javax libraries still hanging around, but not needed